doc/changes/devel/12300.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bug where an event that occurred only once was excluded in :func:`mne.io.read_raw_egi`, by :newcontrib:`Ping-Keng Jao`.
```

Comment:
was this incorporated from a different (abandoned) PR?

Reply:
> was this incorporated from a different (abandoned) PR?

Yep, see `closes` list above has https://github.com/mne-tools/mne-python/pull/11674 and I `cherry-pick`'ed the commits from it